modules/packages/Curl.chpl

```diff
@@ -118,6 +118,7 @@ module Curl {
   require "curl/curl.h";
   require "-lcurl";
 
+  extern proc curl_slist_append(csl: c_ptr(curl_slist), char: c_string) : c_ptr(curl_slist);
```

Comment:
Could you add `curl_slist_free_all()` too, so that `slist.free()` doesn't hit the same problem?

I think the extern procs should be added with the other curl extern procs, around line 347, or towards the end of the list around line 404.

Reply:
@cassella Whoops, sorry, will rock this out tomorrow!

Reply:
@cassella just added and pushed to this PR